otcextensions/sdk/dms/v1/group.py

```diff
@@ -9,28 +9,27 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-from openstack import _log
+from openstack import exceptions
 from openstack import resource
-# from openstack import utils
+from openstack import utils
 
 from otcextensions.sdk.dms.v1 import _base
 
-_logger = _log.setup_logging('openstack')
-
 
 class Group(_base.Resource):
 
     # NOTE: we are not interested in the also returned short queue info
     resources_key = 'groups'
-
     base_path = 'queues/%(queue_id)s/groups'
+
     # capabilities
     allow_create = True
     allow_list = True
     allow_delete = True
 
     _query_mapping = resource.QueryParameters(
-        'include_deadletter'
+        'include_deadletter', 'page_size', 'current_page'
+        # 'include_messages_num'
```

Comment:
why include_messages_num is commented out?
Not yet supported?

Reply:
because it is not properly documented (present in URL path, missing in the description)